### DISCUSSION TITLE
Error ' Not enough degree of freedom'

### DISCUSSION THREAD

**Discussion by oaldaify** [casadi/casadi#3154] [Q&A]
I believe the issue is caused by the equality constraint in the loop, any suggestions on how to fix it.
![1](https://user-images.githubusercontent.com/130773523/234885751-79e3f21a-8432-4dcf-a69d-8f16bee709ff.jpg)
![2](https://user-images.githubusercontent.com/130773523/234885769-540ad2a7-b23a-4431-ab83-ff89976e91b4.jpg)
![3](https://user-images.githubusercontent.com/130773523/234885798-26b33ab9-845f-472c-aca3-d79ce76eb20e.jpg)
![4](https://user-images.githubusercontent.com/130773523/234885835-f9599a7a-7a01-4d63-8b30-408d9d56d22b.jpg)
![5](https://user-images.githubusercontent.com/130773523/234885870-297763f2-bf67-4260-b480-7abeba54c374.jpg)
![6](https://user-images.githubusercontent.com/130773523/234885912-9bb906fd-d97d-4c74-9eb2-df54cd812f2b.jpg)

**Comment by brunoml5** [comment]
Hi! Please share the code in text, not in images, so that other people can run it. Otherwise, it will be very unlikely that somebody will help you. Also, consider using [Github Gist](https://gist.github.com/).
